<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>873</width>
    <height>504</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>873</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>Setting</string>
    </property>
    <widget class="QMenu" name="menuCommunicateTool">
     <property name="title">
      <string>CommunicateTool</string>
     </property>
     <addaction name="actionSerial_Port_2"/>
     <addaction name="actionTCP_Port_2"/>
     <addaction name="actionDebug_Port_2"/>
    </widget>
    <addaction name="menuCommunicateTool"/>
    <addaction name="actionSerial_Port"/>
    <addaction name="actionTCP_Port"/>
    <addaction name="actionDebug_Port"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuConnect">
    <property name="title">
     <string>Connect</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionStruct_Window"/>
    <addaction name="actionSend_Window"/>
    <addaction name="actionRece_Window"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHow_to_use"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuSetting"/>
   <addaction name="menuConnect"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionSerial_Port">
   <property name="icon">
    <iconset resource="images/res.qrc">
     <normaloff>:/image/CONTROLPANEL.ICO</normaloff>:/image/CONTROLPANEL.ICO</iconset>
   </property>
   <property name="text">
    <string>Serial Port</string>
   </property>
  </action>
  <action name="actionTCP_Port">
   <property name="icon">
    <iconset resource="images/res.qrc">
     <normaloff>:/image/CONTROLPANEL.ICO</normaloff>:/image/CONTROLPANEL.ICO</iconset>
   </property>
   <property name="text">
    <string>TCP Port</string>
   </property>
  </action>
  <action name="actionDebug_Port">
   <property name="icon">
    <iconset resource="images/res.qrc">
     <normaloff>:/image/CONTROLPANEL.ICO</normaloff>:/image/CONTROLPANEL.ICO</iconset>
   </property>
   <property name="text">
    <string>Debug Port</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="images/res.qrc">
     <normaloff>:/image/SEA MONSTER.ICO</normaloff>:/image/SEA MONSTER.ICO</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset resource="images/res.qrc">
     <normaloff>:/image/DOWNLOADS567.ICO</normaloff>:/image/DOWNLOADS567.ICO</iconset>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionStruct_Window">
   <property name="icon">
    <iconset resource="images/res.qrc">
     <normaloff>:/image/ACDSEE2.ICO</normaloff>:/image/ACDSEE2.ICO</iconset>
   </property>
   <property name="text">
    <string>Struct Window</string>
   </property>
  </action>
  <action name="actionSend_Window">
   <property name="icon">
    <iconset resource="images/res.qrc">
     <normaloff>:/image/ACDSEE2.ICO</normaloff>:/image/ACDSEE2.ICO</iconset>
   </property>
   <property name="text">
    <string>Send Window</string>
   </property>
  </action>
  <action name="actionRece_Window">
   <property name="icon">
    <iconset resource="images/res.qrc">
     <normaloff>:/image/ACDSEE2.ICO</normaloff>:/image/ACDSEE2.ICO</iconset>
   </property>
   <property name="text">
    <string>Rece Window</string>
   </property>
  </action>
  <action name="actionHow_to_use">
   <property name="icon">
    <iconset resource="images/res.qrc">
     <normaloff>:/image/ADDRESS BOOK.ICO</normaloff>:/image/ADDRESS BOOK.ICO</iconset>
   </property>
   <property name="text">
    <string>How to use</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="images/res.qrc">
     <normaloff>:/image/ADOBE PHOTOSHOP CS.ICO</normaloff>:/image/ADOBE PHOTOSHOP CS.ICO</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSerial_Port_2">
   <property name="icon">
    <iconset resource="images/res.qrc">
     <normaloff>:/image/SEA MONSTER.ICO</normaloff>:/image/SEA MONSTER.ICO</iconset>
   </property>
   <property name="text">
    <string>Serial Port</string>
   </property>
  </action>
  <action name="actionTCP_Port_2">
   <property name="icon">
    <iconset resource="images/res.qrc">
     <normaloff>:/image/SEA MONSTER.ICO</normaloff>:/image/SEA MONSTER.ICO</iconset>
   </property>
   <property name="text">
    <string>TCP Port</string>
   </property>
  </action>
  <action name="actionDebug_Port_2">
   <property name="icon">
    <iconset resource="images/res.qrc">
     <normaloff>:/image/Aqua Ball Green.ico</normaloff>:/image/Aqua Ball Green.ico</iconset>
   </property>
   <property name="text">
    <string>Debug Port</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="images/res.qrc">
     <normaloff>:/image/Aqua Ball Red.ico</normaloff>:/image/Aqua Ball Red.ico</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images/res.qrc"/>
 </resources>
 <connections/>
</ui>
